src/node_api/core/routes.rs

```diff
@@ -264,94 +217,166 @@ impl Client {
+        self.node_manager.get_request(path, None, DEFAULT_API_TIMEOUT).await
+    }
+
+    /// Returns the message that was included in the ledger for a given TransactionId.
+    /// GET /api/v2/transactions/{transactionId}/included-message
+    pub async fn get_included_message(&self, transaction_id: &TransactionId) -> Result<Message> {
+        let path = &format!("api/v2/transactions/{}/included-message", transaction_id);
+
+        let resp = self
+            .node_manager
+            .get_request::<MessageResponse>(path, None, self.get_timeout())
+            .await?;
 
-        Ok(resp)
+        Ok(Message::try_from(&resp.0)?)
     }
 
-    /// Get all stored receipts.
-    /// GET /api/v2/receipts endpoint
-    pub async fn get_receipts(&self) -> Result<Vec<ReceiptDto>> {
-        let path = &"api/v2/receipts";
+    // Milestones routes.
 
-        let resp: ReceiptsResponse = self.node_manager.get_request(path, None, DEFAULT_API_TIMEOUT).await?;
+    /// Gets the milestone by the given milestone id.
+    /// GET /api/v2/milestones/{milestoneId}
+    pub async fn get_milestone_by_id(&self, milestone_id: &MilestoneId) -> Result<MilestoneResponse> {
+        let path = &format!("api/v2/milestones/{}", milestone_id);
 
-        Ok(resp.receipts)
+        self.node_manager.get_request(path, None, self.get_timeout()).await
     }
 
-    /// Get the receipts by the given milestone index.
-    /// GET /api/v2/receipts/{migratedAt} endpoint
-    pub async fn get_receipts_migrated_at(&self, milestone_index: u32) -> Result<Vec<ReceiptDto>> {
-        let path = &format!("api/v2/receipts/{}", milestone_index);
+    /// Gets the milestone by the given milestone id.
+    /// GET /api/v2/milestones/{milestoneId}
+    pub async fn get_milestone_by_id_raw(&self, milestone_id: &MilestoneId) -> Result<Vec<u8>> {
+        let path = &format!("api/v2/milestones/{}", milestone_id);
 
-        let resp: ReceiptsResponse = self.node_manager.get_request(path, None, DEFAULT_API_TIMEOUT).await?;
+        self.node_manager
+            .get_request_bytes(path, None, self.get_timeout())
+            .await
+    }
 
-        Ok(resp.receipts)
+    /// Gets all UTXO changes of a milestone by its milestone id.
+    /// GET /api/v2/milestones/{milestoneId}/utxo-changes
+    pub async fn get_utxo_changes_by_id(&self, milestone_id: &MilestoneId) -> Result<UtxoChangesResponse> {
+        let path = &format!("api/v2/milestones/{}/utxo-changes", milestone_id);
+
+        self.node_manager.get_request(path, None, self.get_timeout()).await
+    }
+
+    /// Gets the milestone by the given milestone index.
+    /// GET /api/v2/milestones/{index}
+    pub async fn get_milestone_by_index(&self, index: u32) -> Result<MilestoneResponse> {
+        let path = &format!("api/v2/milestones/by-index/{}", index);
+
+        self.node_manager.get_request(path, None, self.get_timeout()).await
+    }
```

Comment:
Shouldn't the order be changed so it's first the `get_milestone...` routes and then `get_utxo_changes...`?

Reply:
I'm following the order of the TIP